src/rgw/rgw_sal_motr.cc

```diff
@@ -67,6 +68,89 @@ static unsigned roundup(unsigned x, unsigned by)
   return ((x - 1) / by + 1) * by;
 }
 
+std::string base62_encode(uint64_t value, size_t pad)
+{
+  // Integer to Base62 encoding table. Characters are sorted in
+  // lexicographical order, which makes the encoded result
+  // also sortable in the same way as the integer source.
+  constexpr std::array<char, 62> base62_chars{
```

Comment:
Ideally this base62 encoding & related code should be in some str utility classes in RGW somewhere, however since we try to limit our code to SAL layer, to avoid merge issues with Ceph RGW later, i understand we are keeping in Motr layer itself.
Still, @ujjwalpl @bkirunge7 is that ok if we start thinking of changes beyond SAL layer and with right justification, can ask Ceph to adopt it.

Reply:
@sjain09 @sachinpunadikar any comments here?

Reply:
If we want to add code beyond SAL layer which is specific to motr, we can create a store specific directory at `src/rgw/store/` and add helper files there. Please check [dbstore](https://github.com/Seagate/cortx-rgw/tree/main/src/rgw/store/dbstore) for reference.
Any changes in other files will first require discussion with CEPH developers.

Reply:
yeah this is common utility function in fact, not store specific. But thanks for sharing dbstore example, we can very well create such for motr later.

> Any changes in other files will first require discussion with CEPH developers.

Am aware, thats why things are being crunched in SAL layer